pkg/controller/build/monitor_routine.go

```diff
@@ -75,6 +75,8 @@ func (action *monitorRoutineAction) Handle(ctx context.Context, build *v1.Build)
 		}
 		// Start the build asynchronously to avoid blocking the reconciliation loop
 		routines.Store(build.Name, true)
+
+		// nolint: contextcheck
```

Comment:
not sure if the handler should be passed to the goroutin

Reply:
The build context should not be passed as the build performs asynchronously to the Build CR reconciliation context.